fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/foundationdb/OnlineIndexer.java

```diff
@@ -1490,6 +1514,17 @@ public Builder setIndexingPolicy(@Nullable final IndexingPolicy indexingPolicy)
             return this;
         }
 
+        /**
+         * Add a {@link ScrubbingPolicy} policy. If set, this policy will enforce index scrubbing (instead of index
```

Comment:
Is there anything in the documentation about what index scrubbing actually is?

Reply:
I suppose that "Which verifying a readable index's validity" isn't enough:) 
Shall I add more detailed description about scrubbing inside this comment?

Reply:
Added more.